### ISSUE TITLE
`getNumberOfPayouts()` no revert for non existent `processId`

### ISSUE THREAD

**Issue by rapidddenis** [etherisc/gif-contracts#108]
https://github.com/etherisc/gif-contracts/blob/b58fd27a34452e062d10b175f3bdd7ee16b9d1a7/contracts/modules/PolicyController.sol#L485

all manually defined getters are reverting if id is non existent.

**Comment by matthiaszimmermann** [comment]
true, we might want to consider adding a `payoutsCount` to `IPolicy.Policy` and get rid of `payoutCount` mapping in the `PolicyController`. this would result in the unification of the approach for the getters.

needs some further analysis on how to best refactor.
